.github/actions/openziti-tunnel-build-action/ubuntu-22.04/Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN apt-get -y install \
         doxygen \
         git \
         graphviz \
+        libssl-dev \
```

Comment:
rocky 9 also provides openssl 3, with package `openssl-devel`

Reply:
Cool, yes. I was contemplating doing the same for RH9, too. I'll see if I can get that one to build and run and validate the expected links.